libs/datamodel/connectors/dml/src/model.rs

```diff
@@ -315,7 +314,7 @@ impl Model {
                 .find_scalar_field(name)
                 .or_else(|| model.find_scalar_field(name.to_lowercase().as_ref()))
             {
-                Some(f) => f.field_type == FieldType::Base(ScalarType::DateTime, None),
```

Comment:
potentially bug-fixing, that check wouldn't match `FieldType::NativeType(ScalarType::DateTime, _)`